stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -60,24 +60,15 @@ def assign_quants(self):
         move.mapped('picking_id')._compute_state()
         return {}
 
-    @api.model
-    def _domain_for_available_quants(self, move):
-        return [
-            ('location_id', 'child_of', move.location_id.id),
-            ('product_id', '=', move.product_id.id),
-            ('quantity', '>', 0),
-        ]
-
-    @api.model
-    def _get_available_quants(self, move):
-        domain = self._domain_for_available_quants(move)
-        return self.env['stock.quant'].search(domain)
-
     @api.model
     def default_get(self, fields):
         res = super(AssignManualQuants, self).default_get(fields)
         move = self.env['stock.move'].browse(self.env.context['active_id'])
-        available_quants = self._get_available_quants(move)
+        available_quants = self.env['stock.quant'].search([
+            ('location_id', 'child_of', move.location_id.id),
+            ('product_id', '=', move.product_id.id),
+            ('quantity', '>', 0),
+        ])
```

Comment:
This is a regression from just overriding the module with the latest one from the OCA repo.

ref. https://github.com/OCA/stock-logistics-warehouse/pull/1064